src/ui_fsmenu/main.cc

```diff
@@ -202,7 +204,13 @@ MainMenu::MainMenu(const bool skip_init)
 	singleplayer_.selected.connect([this]() { action(singleplayer_.get_selected()); });
 	multiplayer_.selected.connect([this]() { action(multiplayer_.get_selected()); });
 	editor_.selected.connect([this]() { action(editor_.get_selected()); });
-	replay_.sigclicked.connect([this]() { action(MenuTarget::kReplay); });
+	replay_.sigclicked.connect([this]() {
+		if ((SDL_GetModState() & KMOD_CTRL) != 0) {
+			action(MenuTarget::kReplayLast);
+		} else {
+			action(MenuTarget::kReplay);
+		}
+	});
```

Comment:
I don't really like this, there is no other main menu button that handles Ctrl+Click differently. Turning Replay into a dropdown just for this extra option might be overkill but would still be cleaner IMHO.